services/order.py

```diff
@@ -0,0 +1,46 @@
+from django.utils import timezone
+from db.models import User, Ticket, Order, MovieSession
+from typing import Optional
+from django.db import transaction
+from datetime import datetime
+
+
+def create_order(*, username, tickets, date=None):
+    if not tickets:
+        raise ValueError("Список квитків не може бути порожнім.")
+
+    try:
+        user = User.objects.get(username=username)
+    except User.DoesNotExist:
+        raise ValueError(f"Користувач з ім'ям '{username}' не знайдений.")
+
+    created_at = datetime.strptime(date, "%Y-%m-%d %H:%M") if date else timezone.now()
```

Comment:
Using `datetime.strptime` for parsing dates can lead to errors if the date format is incorrect. Consider adding error handling or validation to ensure the date format is correct.